### PR TITLE
fr-FR: Various improvements

### PR DESCRIPTION
**Pull request by hlabrand**
Hello! I read the whole fr-FR.txt file from beginning to end and fixed a few problems, ranging from typos to missing translations and a few suggested improvements. I have tried to separate them in different commits so it's easier to review. Let me know if you have any questions! Thanks!

### CODE REVIEW

**Comment by tupaschoal** [comment]
Wanna double check @rnmvgr?

File: data/language/fr-FR.txt [hunk 1]
```diff
@@ -3643,7 +3643,7 @@ STR_6529    :Paramètre de palette de couleurs invalide !
 STR_6530    :User Created Expansion Set
 STR_6531    :La machine temporelle
 STR_6532    :Le monde de Katy
-STR_6533    :{WINDOW_COLOUR_2}Indice d’excitation : {BLACK}-{COMMA16}%
+STR_6533    :{WINDOW_COLOUR_2}Indice d'engouement : {BLACK}-{COMMA16}%
```
Comment (rmnvgr) [inline data/language/fr-FR.txt:3646]:
```suggestion
STR_6533    :{WINDOW_COLOUR_2}Indice d’engouement : {BLACK}-{COMMA16}%
```

File: data/language/fr-FR.txt [hunk 2]
```diff
@@ -4433,27 +4433,27 @@ STR_DTLS    :Comme à son jour d’ouverture — 18 mai 1905.{NEWLINE}Par : Aet
 <Mount Vesuvius 1700 A.D. by Katatude for UCES>
 STR_SCNR    :Le mont Vésuve
 STR_PARK    :Le mont Vésuve
-STR_DTLS    :Pompeii and Herculaneum were buried by Mt. Vesuvius in 79 A.D. Visit the excavations and build a park!{NEWLINE}Author: Katatude
+STR_DTLS    :Le mont Vésuve a enseveli Pompéi et Herculanum en 79 av.J.C. Visitez le site de fouilles et construisez un parc !{NEWLINE}Par : Katatude
```
Comment (rmnvgr) [inline data/language/fr-FR.txt:4436]:
```suggestion
STR_DTLS    :Le mont Vésuve a enseveli Pompéi et Herculanum en 79 av. J.-C. Visitez le site de fouilles et construisez un parc !{NEWLINE}Par : Katatude
```

**Review by rmnvgr — CHANGES_REQUESTED** [review]
@hlabrand Thanks a lot! I made a few suggestions to your changes.

**Comment by hlabrand** [comment]
Great suggestions, well spotted! Thank you!

**Review by rmnvgr — APPROVED** [review]
Looks good to me 👍

**Pull request merged**